Classes/Queue/FakeQueue.php

```diff
@@ -76,7 +76,7 @@ public function submit($payload, array $options = [])
     {
         $messageId = Algorithms::generateUUID();
         $message = new Message($messageId, $payload);
-        $commandArguments = [$this->name, base64_encode(serialize($message))];
+        $commandArguments = ['queue' => $this->name, 'messageCacheIdentifier' => base64_encode(serialize($message))];
```

Comment:
Note: This looks broken even in the uchanged version, since the command expects a cache identifier, but this still gives the serialized message…

Reply:
Hah, see https://github.com/Flowpack/jobqueue-common/issues/34